ui/src/loudml/components/AboutPanel.js

```diff
@@ -27,7 +28,10 @@ const AboutPanel = ({
                     It's plug-and-play, so you can use it for other projects too.
                 </p>
                 <p style={AboutTextStyle}>
-                    Visit <a href='http://loudml.io' target='_loudml'>loudml.io</a> to download the free Community package and reap the benefits of ML within days.
+                    Contribute to the open source software development on <a href="https://github.com/regel" target="_github">GitHub</a>.
```

Comment:
Pardon, l'url est: https://github.com/regel/loudml

Reply:
in next commit